src/components/LoginForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+const LoginForm = () => {
+  const [userName, setUserName] = useState("");
+  const [password, setPassword] = useState("");
+
+  return (
+    <form
+      className="my-5 max-w-md mx-auto self-center px-5 rounded-2xl shadow-lg flex flex-col flex-grow justify-center bg-cblue"
+      onSubmit={(ev) => {
+        ev.preventDefault();
+        fetch("/api/auth/Login", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({ userName: userName, password: password }),
+        });
```

Comment:
You're not doing anything with the results. What should happen after logging in?